src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```
@@ -445,7 +445,7 @@
       "path": "BlazorServerWeb-CSharp.csproj"
     }
   ],
-  "defaultName": "WebApplication",
+  "defaultName": "BlazorApp",
```

We don't abbreviate things on user displayed strings
```suggestion
  "defaultName": "BlazorApplication",
```

---

@danroth27 I'll let you suggest the exact phrase here.

---

App is not an abbreviation. It is a modern synonym for application. Examples of this usage include "Apple App Store". Use is "app" is preferred over "application" in the Microsoft Style Guide: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/app-application.

---

Should we update WebApplication to WebApp to then? https://github.com/dotnet/aspnetcore/blob/241b10a43fecc7a08ad875bb4f483db8cc754511/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/.template.config/template.json#L374

---

This change was motivated by the principle of "don't move people's cheese unnecessarily". Blazor apps in VS historically default to the BlazorApp name, but that changed with the move to the new New Project Dialog experience. This PR fixes that and avoids us having to update docs, screen shots, etc. Web apps have historically defaulted to WebApplication1, and I don't see any reason to change that. I brought up the Microsoft Style Guide purely to show that "app" is valid, not that it is always preferred.